### PR TITLE
#2024 Constraints changes, help, junits

### LINKED ISSUE #2024: CUSTOMER - Constraints on OEntities and OActors should be displayed in referencing elements inside the Semantic Browser view
Constraints applied on the selected element should be displayed in the Semantic Browser (left-hand part: Constraining Elements).
They are not displayed on Operational Entities. Same problem for Operational Actors (Operational Entity [Actor]).

Note that they are correctly displayed on System Actors, the System, Operational Activities and Interactions.

### PR DESCRIPTION
**Pull request by claudiasima-ro**
Fixes #2024

### CODE REVIEW

**Comment by pdulth** [comment]
Can you do a rebase, i have squash both previous commits to one.

**Comment by pdulth** [comment]
I didn't understood why this commit couldn't have been merged in the previous PR though.

**Review by pdulth — CHANGES_REQUESTED** [review]

File: doc/plugins/org.polarsys.capella.validation.doc/html/Validation Rules/design/well-formedness/data/ValidationRules.mediawiki [hunk 1]
```diff
@@ -337,7 +337,7 @@ If an Association is on SystemArchitecture layer. It should not associate with a
 !|[[Image:../../../../Images/error.gif|ERROR]]
 |DWF_D_52 - Type Of Default And Null Value For Property 
 |-
-| colspan="2"|For a property, type of the default (resp. null) value shall be Undefined (implicit typing) or the property�s datatype or a super type of the property�s datatype
+| colspan="2"|For a property, type of the default (resp. null) value shall be Undefined (implicit typing) or the property�s datatype or a super type of the property�s datatype
```
[the remark this answers is not in the document]
Comment (claudiasima-ro) [reply doc/plugins/org.polarsys.capella.validation.doc/html/Validation Rules/design/well-formedness/data/ValidationRules.mediawiki]: Done

**Review by georgiana-ecobici — APPROVED** [review]

**Review by pdulth — APPROVED** [review]

File: tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_58.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import org.polarsys.capella.test.framework.api.OracleDefinition;
+import org.polarsys.capella.test.validation.rules.ju.testcases.AbstractRulesOnDesignTest;
+
+/**
+ * test on Rule_DWF_D_58: This rule ensures that in monopart mode a constraint constrains the Component and not the Part or PartDeploymentLink.
+ * @generated
+ */
+public class Rule_DWF_D_58 extends AbstractRulesOnDesignTest {
+
+  public static final String ENUMERATION_LITERAL_1 = "8ffa834f-648f-46aa-9e7c-abe35dc0289d";
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_58.java:29]: can you rename from enumeration_literal to constraints?

File: tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_58.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  public static final String ENUMERATION_LITERAL_2 = "ac316f95-96e0-4072-ae67-41732f1842d1";
+  
+  /**
+   * @see org.polarsys.capella.test.validation.rules.ju.testcases.ValidationRuleTestCase#getTargetedEClass()
+   * @generated
+   */
+  protected EClass getTargetedEClass() {
+    return DatavaluePackage.Literals.ENUMERATION_LITERAL;
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_58.java:37]: Constraint no?

File: tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_59.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import org.polarsys.capella.test.framework.api.OracleDefinition;
+import org.polarsys.capella.test.validation.rules.ju.testcases.AbstractRulesOnDesignTest;
+
+/**
+ * test on Rule_DWF_D_59: This rule ensures that in monopart mode a constraint constrains the Component and not the Part or PartDeploymentLink.
+ * @generated
+ */
+public class Rule_DWF_D_59 extends AbstractRulesOnDesignTest {
+
+  public static final String ENUMERATION_LITERAL_1 = "ac316f95-96e0-4072-ae67-41732f1842d1";
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.validation.rules.ju/src/org/polarsys/capella/test/validation/rules/ju/testcases/dwf_d/Rule_DWF_D_59.java:29]: same here

**Review by pdulth — CHANGES_REQUESTED** [review]

**Review by georgiana-ecobici — APPROVED** [review]

**Review by georgiana-ecobici — APPROVED** [review]

**Review by pdulth — APPROVED** [review]

**Pull request merged**